### ISSUE TITLE
Consider adding notes about unsafe usages to README

### ISSUE THREAD

**Issue by jchv** [samuela/rustybox#4]
As users have pointed out on HN, since this is a direct conversion from C, there’s a lot of usages of `unsafe`. I think it would be nice to denote this in two ways:

- A statement exclaiming such somewhere near the top, like “This is still a work in progress and contains memory-unsafe code as translated from C.”

- A bullet point in the contributing section about reducing `unsafe` usages, as imaginably that would be a useful goal.

I am interested in trying to contribute and if I do this may be a focus area for me.

Cheers.

**Comment by samuela** [comment]
Hey @jchv, thanks for pointing that out! Sadly there are still a bunch of `unsafe`s lurking. I've added a section to the README that points this out and discusses the state of the project (a3c177677dc8f5c38bf80c47502b6d27a73c8860). I also added a section to CONTRIBUTING.md to address your second point (3a61bee81f98f4b3d8a4710515de7352c8342b13). Hopefully these changes should help to clarify where the project stands, and where the opportunities for refactoring are! 

The ultimate goal is to start from something imperfect but working and continually improve on it everyday.
